core/templates/dev/head/pages/topic_editor/TopicEditorNavbarDirective.js

```diff
@@ -58,7 +58,11 @@ oppia.directive('topicEditorNavbar', [
               function() {
                 $scope.topicRights.markTopicAsPublished();
                 TopicEditorStateService.setTopicRights($scope.topicRights);
-              });
+              }).then(function() {
```

Comment:
These are the errors I am getting locally:
```
/core/templates/dev/head/pages/topic_editor/TopicEditorNavbarDirective.js
  62:1  error  Expected indentation of 14 spaces but found 16  indent
  63:1  error  Expected indentation of 14 spaces but found 16  indent
  64:1  error  Expected indentation of 16 spaces but found 18  indent
```
Please break after `then(` similar to the above `then(` and then indent them:
```javascript
              }).then(
              function() {
                var successToast = 'Topic published.';
                AlertsService.addSuccessMessage(
                  successToast, 1000);
              });
          };
```
This will fix the build.

Reply:
That's odd. I had just advised against this -- since the function() line should be indented due to the open paren before it.

However, I looked at the context and I think I know what's going on. If you look at lines 57-61 there's a open paren at the end of line 57 that gets closed on line 61, causing a mismatch. We should probably break after line 61 instead.

That should ensure that the open/close parens/brackets/braces line up correctly. Would that solve the problem?

```suggestion
              }
            ).then(function() {
```

Reply:
Yup, both of these work. The linter had suggested this way but I thought we needed to break it up after `.then(`, therefore I suggested the former. Understood now, thanks!

Reply:
Done.